docs/getting-started.md

```diff
@@ -231,9 +277,8 @@ git checkout tags/v0.10.1
 
 Within the Firecracker repository root directory:
 
-```bash
-tools/devtool build
-```
+1. on a **x86_64** machine:  ```tools/devtool build```
+1. on an **aarch64** machine: ```tools/devtool build -- --target aarch64-unknown-linux-musl```
```

Comment:
Why isn't musl the default on aarch64?

Reply:
So, cargo will use as default the thing you specify in the build section of the cargo/.config.
Ours looks like this.
```
[build]
target = "x86_64-unknown-linux-musl"
```
So, it will try to build with this target on aarch64. Unfortunately I have not found a way to adjust it based on architecture. I am open to suggestions.
I tried using:
```
[build.'cfg(target_arch = "x86_64")']
target = "x86_64-unknown-linux-musl"

[build.'cfg(target_arch = "aarch64")']
target = "aarch64-unknown-linux-musl"

```
but cargo will ignore them and start considering default target the gnu target even for x86_64 arch so it seems that it breaks even the x86_64 settings.
I am open to get help here.

Reply:
`.cargo/config` doesn't seem to support having multiple targets based on architecture: https://github.com/rust-lang/cargo/issues/3913.